### ISSUE TITLE
Update documentation to include Sensors.ADXL345 if using Renode 1.9 release

### ISSUE THREAD

**Issue by lisaong** [antmicro/litex-vexriscv-tensorflow-lite-demo#17]
Hi, first of all, thanks for the demo!

I tried to get this up and running with Renode, Sensors.ADXL345 is not found:

```
Renode, version 1.9.0.27103 (169a3c85-202003101317)
(monitor) s @litex-vexriscv-tflite.resc
There was an error executing command 'machine LoadPlatformDescription /home/lisa/li
tex-vexriscv-tensorflow-lite-demo/renode/litex-vexriscv-tflite.repl'
Error E04: Could not resolve type: 'Sensors.ADXL345'.
At /home/lisa/litex-vexriscv-tensorflow-lite-demo/renode/litex-vexriscv-tflite.repl
:28:10:
adxl345: Sensors.ADXL345 @ i2c 0x1D
         ^^^^^^^^^^^^^^^^
(machine-0)
```

Turns out the 1.9 version from Renode's Releases page does not contain the Emulation for ADXL345. Rebuilding Renode to latest would solve it, but it's a bit heavyweight just to add 1 sensor.

In case anyone is puzzling over it, here's the fix. Will be great if this can be noted in the Readme:

1. Download the ADXL345 emulator from the Renode Github repository:
```
wget https://raw.githubusercontent.com/renode/renode-infrastructure/master/src/Emulator/Peripherals/Peripherals/Sensors/ADXL345.cs
```

2. Edit the litex-vexriscv-tflite.resc to include the ADXL345.cs:

```
using sysbus

include $ORIGIN/LiteX_I2C_Zephyr.cs
include $ORIGIN/ADXL345.cs
```

**Comment by pgielda** [comment]
Judging from your Renode version you are using a version from 2020.30.10 and ADXL345 was added later than that.

Having said that you are most probably using the last binary release - there will be a new one soon and that one will have ADXL345 in it.

**Comment by lisaong** [comment]
Indeed, I am using the latest binary release that missed the addition of ADXL345 by a couple of days. 

Good to hear that there is a plan to update the binary release of renode. Feel free to close the issue as well.

**Comment by mateusz-holenko** [comment]
@lisaong We released [Renode 1.10](https://github.com/renode/renode/releases/tag/v1.10.1) that includes the ADXL345 model - switching to this release should solve the problem.
